### DISCUSSION TITLE
Dark / light theme switching using stylesheets with persistance

### DISCUSSION THREAD

**Discussion by tonven** [notJust-dev/notjustdev#172] [Ideas]
Dark / light theme switching using stylesheets with persistance

**Comment by Ajmal0197** [comment]
Code:
https://github.com/Ajmal0197/RN_Dev_Codes/tree/main/src/srcThemeing

**Comment by Janssonn91** [comment]
> Code: https://github.com/Ajmal0197/RN_Dev_Codes/tree/main/src/srcThemeing

I am doing same implementation as you. However I have no idea if it's best practice to do it this way. I find it slightly annoying to constantly import useTheme to get hold of my colors. Is there no way to make it global everywhere Or by just importing the colors.

**Reply by Ajmal0197** [reply]
You can use Context API to do so but for the same u will have to import `useContext` in each screen.
